scripts/deploy-federation.sh

```diff
@@ -194,10 +194,14 @@ fi
 # Create a permissive clusterrolebinding to allow federation controllers to run.
 if [[ "${NAMESPACED}" ]]; then
   # TODO(marun) Investigate why cluster-admin is required to view cluster registry clusters in a namespace
-  kubectl -n "${NS}" create rolebinding federation-admin --clusterrole=cluster-admin --serviceaccount="${NS}:default"
+  if ! kubectl -n "${NS}" get rolebinding federation-admin; then
```

Comment:
Are you handling a scenario, where the rolebinding `federation-admin` already exist in the cluster?

Reply:
yes.

Reply:
What does this change have to do with improving darwin support?  Please ensure this PR has a limited and focused scope to make life easier for reviewers and future maintainers.